Makefile

```diff
@@ -145,16 +145,20 @@ run: manifests generate fmt vet ## Run a controller from your host.
 	$(GO_CMD) run ./main.go
 
 # Build the container image
+.PHONY: image-local-build
+image-local-build:
```

Comment:
Also update the documentation https://github.com/kubernetes-sigs/kueue/blob/main/docs/setup/install.md#build-and-install-from-source

Reply:
Sure. Thanks!